js/source/raster_tile_source.js

```diff
@@ -78,7 +81,9 @@ RasterTileSource.prototype = util.inherit(Evented, {
             gl.generateMipmap(gl.TEXTURE_2D);
 
             tile.timeAdded = new Date().getTime();
-            this.map.animationLoop.set(this.style.rasterFadeDuration);
+            if (this.map.style != null) {
+                this.map.animationLoop.set(this.map.style.rasterFadeDuration);
+            }
```

Comment:
I suspect the fix proposed above 👆 will fix this case too.

Without a demonstration of the bug, it is impossible for me to know for sure.